src/pages/home/ReportScreen.js

```diff
@@ -163,7 +164,7 @@ function ReportScreen({
     const screenWrapperStyle = [styles.appContent, styles.flex1, {marginTop: viewportOffsetTop}];
 
     // There are no reportActions at all to display and we are still in the process of loading the next set of actions.
-    const isLoadingInitialReportActions = _.isEmpty(reportActions) && reportMetadata.isLoadingReportActions;
+    const isFirstlyLoadingReportActions = _.isEmpty(reportActions) && reportMetadata.isLoadingInitialReportActions;
```

Comment:
NAB: But I find the name of this variable a bit hard to understand what it is exactly doing, but maybe it's just me so not a blocker

Reply:
I wanted to make sure everything was clear with the same prop from reportMetadata. Will return old naming